src/screens/Exchange/Exchange.js

```diff
@@ -691,12 +730,34 @@ class ExchangeScreen extends React.Component<Props, State> {
             </CardInnerRow>
           </CardRow>
           <CardRow>
+            {!!isFiat &&
+            <CardColumn style={{ flex: 1 }}>
+              <CardText label>Fees Total</CardText>
+              <View style={{ flexDirection: 'row' }}>
+                <CardText>{feeAmount !== '' ? `${feeAmount + extraFeeAmount} ${fromAssetCode}` : 'Check'}</CardText>
+              </View>
+            </CardColumn>
+            }
+            {!isFiat &&
             <CardColumn style={{ flex: 1 }}>
               <CardText label>Available</CardText>
               <View style={{ flexDirection: 'row' }}>
                 <CardText>{available}</CardText>
               </View>
             </CardColumn>
+            }
+            {!!isFiat &&
+            <CardColumn>
+              <Button
+                title={isTakeOfferPressed ? '' : `${quoteCurrencyAmount} ${toAssetCode}`}
```

Comment:
format the value as there might be many decimals? use the formatting methods for other exchange cards

Reply:
you can use `formatAmountDisplay`

Reply:
927faae